src/ElmFormat/Render/Box.hs

```diff
@@ -394,15 +417,15 @@ formatImport aimport =
                           ( name', Just as', Just exposing' ) ->
                             stack1
                               [ line $ keyword "import"
-                              , indent name'
-                              , indent $ indent as'
-                              , indent $ indent exposing'
+                              , indent $ name'
+                              , indent $ indent $ as'
+                              , indent $ indent $ exposing'
```

Comment:
These new `$`s aren't necessary, as is the one on 428. Haskell `$` is Elm's `<|`.